src/ray/raylet/worker_pool.h

```diff
@@ -385,7 +436,7 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
     /// The commands and arguments used to start the worker process
     std::vector<std::string> worker_command;
     /// The pool of dedicated workers for actor creation tasks
-    /// with prefix or suffix worker command.
+    /// with dynamic worker options (prefix or suffix worker command.)
```

Comment:
Isn't it for dynamic worker options + other stuff like runtime env? Or am I missing something?

Reply:
It's just for dynamic_options -- let me remove dynamic options from the WorkerCacheKey to make the separation even clearer, as there's no reason for it to be there now.